app/notifiers/publishers/job_application_data_expiry_notifier.rb

```diff
@@ -0,0 +1,36 @@
+class Publishers::JobApplicationDataExpiryNotifier < Noticed::Event
+  deliver_by :database
+  deliver_by :email, mailer: "Publishers::JobApplicationDataExpiryMailer", method: :job_application_data_expiry
+  delegate :created_at, to: :record
+  param :vacancy, :publisher
+
+  notification_methods do
```

Comment:
As of v2 we need to wrap these methods in the notifications_method block to make them available on the notification, rather than the event of the same name